iot/api/CloudIotSample/Program.cs

```diff
@@ -1376,6 +1491,209 @@ public static object ClearRegistry(string projectId, string cloudRegion, string
         }
         //[END iot_clear_registry]
 
+
+        //[START iot_attach_device]
+        public static object AttachDevice(MqttClient client, string deviceId)
+        {
+            var attachTopic = $"/devices/{deviceId}/attach";
+            Console.WriteLine("Attaching: {0}", attachTopic);
+            var attachPayLoad = "{}";
+            var BinaryData = Encoding.UTF8.GetBytes(attachPayLoad);
+            client.Publish(attachTopic, BinaryData, MqttMsgBase.QOS_LEVEL_AT_LEAST_ONCE, true);
+
+            Console.WriteLine("Waiting for device to attach.");
+            System.Threading.Thread.Sleep(2000);
+            return 0;
+        }
+        //[END iot_attach_device]
+
+        //[START iot_detach_device]
+        public static object DetachDevice(MqttClient client, string deviceId)
```

Comment:
The sample should demonstrate how to create the client.

Reply:
These changes have been made in previous PR which is still under review.